src/guide/migration/custom-elements-interop.md

```diff
@@ -7,7 +7,7 @@ badges:
 
 ## Overview
 
-- **BREAKING:** Custom elements whitelisting is now performed during template compilation, and should be configured via compiler options instead of runtime config.
+- **BREAKING:** Custom elements allow listing is now performed during template compilation, and should be configured via compiler options instead of runtime config.
```

Comment:
```suggestion
- **BREAKING:** The checks to determine whether tags should be treated as custom elements are now performed during template compilation, and should be configured via compiler options instead of runtime config.
```